ytmusicapi/navigation.py

```diff
@@ -60,6 +60,7 @@
 TASTE_PROFILE_ITEMS = ["contents", "tastebuilderRenderer", "contents"]
 TASTE_PROFILE_ARTIST = ["title", "runs"]
 SECTION_LIST_CONTINUATION = ['continuationContents', 'sectionListContinuation']
+MENU_PLAYLIST_ID = MENU_ITEMS + [0 + 'menuNavigationItemRenderer'] + NAVIGATION_WATCH_PLAYLIST_ID
```

Comment:
This part is not actually working. Please test your changes before submission.

`TypeError: unsupported operand type(s) for +: 'int' and 'str'`

Reply:
Yes, apologies, I wanted to prettify my change and should have taken the time to check it.